modules/olm-installing-from-operatorhub-using-web-console.adoc

```diff
@@ -81,17 +90,12 @@ endif::[]
 .. For clusters on cloud providers with token authentication enabled:
 +
 --
-* If the cluster uses {aws-short} {sts-short} (*STS Mode* in the web console), enter the Amazon Resource Name (ARN) of the AWS IAM role of your service account in the *role ARN* field.
-+
-image::oadp-install-operator-role-arn.png[Entering the ARN]
```

Comment:
This screenshot was introduced as a result of a review request in https://github.com/openshift/openshift-docs/pull/66903#pullrequestreview-1731973852, however I'm removing it now for a few reasons:

* Primarily, we have style guidelines to avoid screenshots in general (https://redhat-documentation.github.io/supplementary-style-guide/#screenshots) and having the screenshot isn't required here.
* The matching Entra ID-related bullet does not have a matching screenshot, so it feels/looks out of place.
* This is the only image/screenshot in the entire assembly, so it feels even more out of place.